packages/smb/src/types/global.d.ts

```diff
@@ -6,6 +6,7 @@ export type Config = {
   googleAnalyticsKey: string
   cognitoOAuthUrl: string
   cognitoUserPoolId: string
+  accessToken: string
```

Comment:
could we use localstorage instead of add it to config ?

Reply:
So I agree with @tanphamhaiduong here - I see why you have done this now however, configs by definition are static constants whereas the accessToken is stateful so not appropriate. Session storage seems to make most  sense :-)